src/Stratis.Bitcoin.Features.Consensus/Checkpoints.cs

```diff
@@ -134,46 +134,54 @@ public class Checkpoints : ICheckpoints
         };
 
         /// <summary>Checkpoints for the specific instance of the class and its network.</summary>
-        private readonly Dictionary<int, CheckpointInfo> checkpoints;
+        private readonly Dictionary<int, CheckpointInfo> networkSpecificCheckpoints;
+
+        /// <summary>Empty list of checkpoints for the specific instance of the class in case the use of checkpoints is disabled in the consensus settings.</summary>
+        private readonly Dictionary<int, CheckpointInfo> emptyCheckpoints = new Dictionary<int, CheckpointInfo>();
```

Comment:
We usually avoid instantiating objects outside of the ctor.
Also it seems this object is never filled (empty, I get it), and then only returned in `GetCheckpoints()`. Is it really necessary to have?

Reply:
I could just return a new empty one instead.